src/Eto.Mac/Forms/Controls/WKWebViewHandler.cs

```diff
@@ -10,7 +10,7 @@ public class WKWebViewHandler : MacView<wk.WKWebView, WebView, WebView.ICallback
 	{
 		public override NSView ContainerControl { get { return Control; } }
 
-		public wk.WKWebViewConfiguration Configuration { get; set; } = new wk.WKWebViewConfiguration();
+		public wk.WKWebViewConfiguration Configuration { get => Control.Configuration; }
```

Comment:
This might break some scenarios where the configuration is set by a derived handler or custom style.  I think we'll still want a setter here for those cases.

Reply:
Good point. I've reinstated this in https://github.com/picoe/Eto/pull/2675/commits/e4bedc3251db1ce6ff086bdab1043def5aba2497 and used `CreateConfiguration` to create a default value in the constructor.